sema4ai/src/sema4ai_code/robocorp_language_server.py

```diff
@@ -5,20 +5,26 @@
 from base64 import b64encode
 from collections.abc import Iterator, Sequence
 from functools import partial
-from pathlib import Path
-from typing import Any
+from pathlib import Path, WindowsPath
```

Comment:
Remove imports added here which shouldn't be needed after using the `uris` module for the uri->filesystem path conversion.